app/views/content/non-uk-teachers/teach-in-england-if-you-trained-overseas.md

```diff
@@ -86,36 +91,31 @@ If you have a teaching qualification from the following countries and regions, y
 * Ukraine
 * the USA
 
-In addition, if you have a subject specialism in languages, mathematics or science and a teaching qualification to teach to teach 11 to 16 year olds from the following countries, you will be able to use the service to check if you meet the requirements for QTS: 
+In addition, if you have a subject specialism in languages, mathematics or science and a teaching qualification to teach to teach 11 to 16 year olds from the following countries, you will be able to use the service to check if you meet the requirements for QTS and apply: 
 
 * Ghana
 * India
 * Jamaica
 * Nigeria
 * Singapore
 * South Africa
-* Zimbabwe
-
-You will not have to pay a fee or undergo further training. However, to be awarded QTS you will need to meet certain criteria. [Learn more about the qualifications and experience you'll need to get QTS](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers). 
 
-<p class="call-to-action__action">
-  <a href="https://apply-for-qts-in-england.education.gov.uk/eligibility/start">Apply for QTS <span>in England</span></a>
-</p>
+ Visit ['Awarding qualified teacher status to overseas teachers'](https://www.gov.uk/government/publications/awarding-qualified-teacher-status-to-overseas-teachers) to check the eligibility criteria and apply for QTS in England.  
 
+### Other routes to QTS in England 
 
-If you’re not eligible to use the new service, you may be able to:
+Many teachers will not meet the requirements for QTS. If you’re not eligible, you may be able to [find an alternative route to QTS which better suits your circumstances](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk).
 
-* [complete initial teacher training in England](/non-uk-teachers/train-to-teach-in-england-as-an-international-student)
-* [apply for international qualified teacher status (iQTS)](/non-uk-teachers/international-qualified-teacher-status)
 
+### Find a teaching job in England  
 
-## Find a teaching job
+To find a teaching job, you must apply to indvidual schools (not the Department for Education). If you need a Skilled Worker visa, you must find a school which can sponsor your visa.
```

Comment:
you must find a school that ...? Not sure if it's which or that?

Reply:
I think you are right it's that – I can only ever tell by seeing if you can put a comma before which and I don't think you can!